.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,20 @@
+name: Check PR Title
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - ready_for_review
+      - reopened
+
+jobs:
+  check-pr-title:
+    name: Check PR title
+    timeout-minutes: 10
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: blumilksoftware/action-pr-title@e05fc76a1cc45b33644f1de51218be43ac121dd0 # v1.2.0
+        
```

Comment:
```suggestion

Reply:
Changed

Reply:
sorry my mistake - please add a blank line everywhere under the code where you have a red circle